jax/core.py

```diff
@@ -317,6 +318,19 @@ def lift(self, tracer):
   def sublift(self, tracer):
     assert False
 
+  def process_primitive(self, primitive, tracers, params):
+    """Processes a primitive
```

Comment:
I don't think docstrings like this add much value?

Reply:
I do want to document Trace and Tracer. I believe there is value in this effort, but we do need more meaningful docstring than this one. I will delete this for now.